nidirect_related_content/nidirect_related_content.module

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * @file
+ * Contains nidirect_related_content.module.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\views\ViewExecutable;
+use Drupal\Core\Block\BlockPluginInterface;
+
+/**
+ * Implements hook_views_pre_build().
+ */
+function nidirect_related_content_views_pre_build(ViewExecutable $view) {
+  if ($view->id() != 'articles_by_term') {
+    return;
+  }
+
+  if ($view->current_display != 'block_1') {
```

Comment:
I'd be tempted to put this in as an && condition to the if statement above as a blanket condition for the correct View and Display.

Reply:
Probably more concise, agreed. I'll crunch them down a bit as suggested 👍